x-pack/plugins/uptime/public/components/monitor/ping_list/ping_list.tsx

```diff
@@ -200,6 +207,15 @@ export const PingList = () => {
         itemId="docId"
         itemIdToExpandedRowMap={expandedRows}
         pagination={pagination}
+        noItemsMessage={
+          loading
+            ? i18n.translate('xpack.uptime.pingList.pingsLoadingMesssage', {
```

Comment:
I'm not sure who determines what the appropriate content is in these cases. Please let me know if this default content is appropriate or need to be run past product.

Reply:
I think `'Loading pings'` might accomplish the same thing. @andrewvc any thought on this copy?

Reply:
On the ticket Paul suggested the more generic `Loading history..` could work well, since this is a history table.